### PR TITLE
Stats: Update traffic section's background color

### PR DESCRIPTION
**Pull request by wilmersondasilva**
#### Proposed Changes

Updates the background colors for the traffic section in Stats.

Before:
image

After:
image

#### Testing Instructions

1. Visit the Calypso Live link.
2. Visit the Stats page.
4. Confirm it matches the screenshot above.
5. Confirm it behaves appropriately on smaller screen sizes.

#### Pre-merge Checklist

- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

Related to #69953

### CODE REVIEW

**Comment by wilmersondasilva** [comment]
@keoshi, I would appreciate it if you take a look at this.

**Comment by keoshi** [comment]
@wilmersondasilva Apologies for the delays but I was never notified of this one 🤔  Going forward please ping me for a review since that seems to alert me differently. Thank you!

First off, this looks great! It's amazing how a subtle grey changes the entire page. Noticed a few things, but most are minor.

#### Content width

[This `fullWidthLayout` change](https://github.com/Automattic/wp-calypso/pull/70093/files#diff-ea5f237b7989be993ecfab2290e70ff69102850504ef05474ce36ff414a5afacR459) grows the layout from `1040px` to `1244px`, when the proposed was `1224px` (20px less). Maybe this difference is coming from the `calc()`?

- [x] It's a small amount, but could we slightly reduce the total width to match the `1224px value`?

#### Chart container background color

image

- [x] Remove `background-color: var(--color-surface)` from `.chart` element, so it blends with the new page background.

#### Jetpack logo set in white bg

image

- [x] Could we bring the Jetpack logo into the same background color as the rest of the page?

Super nice work, @wilmersondasilva — thanks again and thanks for adding the subtle inner shadows too!

**Comment by wilmersondasilva** [comment]
Thanks for the feedback, @keoshi.

About the `1244px`, it was my fault, actually. But I just fixed it. 

> Remove `background-color: var(--color-surface)` from `.chart` element, so it blends with the new page background.

Regarding this one, what `background-color` should the `.chart` element have? I'll leave it as `#fdfdfd` for now, since it's the same as the parent element.

**Comment by keoshi** [comment]
> Regarding this one, what `background-color` should the `.chart` element have? I'll leave it as `#fdfdfd` for now, since it's the same as the parent element.

I was thinking we could get rid of it entirely, but setting it to the parent's bg also works.

Thank you so much once again!

**Comment by keoshi** [comment]
Oh, one more thing: could only see the new styles in the Traffic page. Should we create follow-up issues/PRs for all other Stats pages: Insights, Ads, and Store?

**Comment by wilmersondasilva** [comment]
> Oh, one more thing: could only see the new styles in the Traffic page. Should we create follow-up issues/PRs for all other Stats pages: Insights, Ads, and Store?

Yes, if I got it right, the idea is to apply these changes only for the Traffic page for now, and then, create a follow-up task to apply it to the other pages

**Comment by wilmersondasilva** [comment]
@keoshi just pushed all the changes you suggested.

**Review by keoshi — APPROVED** [review]
Looks great in my testing. Thanks again, @wilmersondasilva!

File: client/blocks/stats-navigation/style.scss [hunk 1]
```diff
@@ -1,4 +1,6 @@
 .stats-navigation {
+	box-shadow: inset 0 -1px 0 #0000000d;
```
Comment (jsnmoon) [inline client/blocks/stats-navigation/style.scss:2]: Let's define this color as a variable and re-use it across the Stats codebase, similar to what we already do within [the horizontal bar list component](https://github.com/Automattic/wp-calypso/blob/trunk/packages/components/src/horizontal-bar-list/_stats-variables.scss).

File: client/my-sites/stats/modernized-chart-tabs-styles.scss [hunk 2]
```diff
@@ -3,9 +3,13 @@
...
 $yearBarChartBorderRadius: 4px;
 
+.stats > .stats--new-main-chart {
+	padding-top: 48px;
+}
+
 // For feature `stats/new-main-chart`
 .stats--new-main-chart {
-	background-color: var(--color-surface);
+	background-color: #fdfdfd;
```
Comment (jsnmoon) [inline client/my-sites/stats/modernized-chart-tabs-styles.scss:12]: Let's also refactor this color as a variable for reuse.

File: client/my-sites/stats/style.scss [hunk 3]
```diff
@@ -224,6 +248,22 @@
 	}
 }
 
+// this overrides the default .layout__content that adds unwanted padding
+.is-section-stats .layout__content,
+.is-section-stats.theme-default .focus-content .layout__content {
+	padding: 79px 0 0 0;
```
Comment (jsnmoon) [inline client/my-sites/stats/style.scss:254]: Could we explain the 79px padding in a comment or roll it into a variable?

File: client/my-sites/stats/style.scss [hunk 4]
```diff
@@ -30,6 +30,30 @@
...
+}
+
+.stats > * {
+	@media ( min-width: 660px ) {
+		padding: 0 32px;
+	}
+
+	// 272 is the --sidebar-width-max value that can't be used in media queries.
+	// 1224 is the max-width for the stats content
+	@media ( min-width: calc(272px + 1224px) ) {
```
Comment (jsnmoon) [inline client/my-sites/stats/style.scss:52]: TIL you can't use CSS variables in media queries, bummer.

**Review by jsnmoon — APPROVED** [review]
Looks good, shipping!

**Pull request merged**